test/handler.test.js

```diff
@@ -432,10 +436,16 @@ function ErrorWithProps(props) {
 util.inherits(ErrorWithProps, Error);
 
 function getExpectedErrorData(err) {
-  // "stack" is a non-enumerable property
-  var data = {stack: err.stack};
-  for (var prop in err) {
-    data[prop] = err[prop];
+  var data = {};
```

Comment:
Can we call `dataBuilder.cloneAllProperties` instead of duplicating this code? I don't remember, was it an intentional decision to duplicate it?

If we go down the route of `dataBuilder.cloneAllProperties`, then obviously we need to modify `dataBuilder` to export this function.